recipe/meta.yaml

```
@@ -1,52 +1,60 @@
 {% set name = "pystache" %}
-{% set version = "0.5.4" %}
-{% set build = 1 %}
-{% set md5 = "485885e67a0f6411d5252e69b20a35ca" %}
+{% set version = "0.6.5" %}
 
 package:
-    name: {{ name }}
-    version: {{ version }}
+  name: {{ name }}
+  version: {{ version }}
 
 source:
-    fn: {{ name }}-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    md5: {{ md5 }}
+  url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9f238d5a06f18843e0d491d8e4e292dc03fed6a54cb0a5c34be37a3faa973174
 
 build:
-    number: {{ build }}
-    script: python setup.py install --single-version-externally-managed --record record.txt
-    noarch: python
-    entry_points:
-        - pystache = pystache.commands.render:main
-        - pystache-test = pystache.commands.test:main
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  entry_points:
+    - pystache = pystache.commands.render:main
+    - pystache-test = pystache.commands.test:main
 
 requirements:
-    build:
-        - python
-        - setuptools
-
-    run:
-        - python
+  host:
+    - python
+    - setuptools
+    - setuptools-scm 7.1.0
+    - wheel
+    - pip
+  run:
+    - python
+    - importlib-metadata >=4.6
```

```suggestion
    - importlib-metadata >=4.6  # [py<310]
```
See https://files.pythonhosted.org/packages/4d/6f/e3d8ec5c636dc69a6dadd27674addc8abf5627c927d3cbcd853b3ba9093d/pystache-0.6.5.tar.gz

---

Thank you!